### PR TITLE
New version: BraveSoftware.BraveBrowser version 98.1.36.106

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/49167)

### CODE REVIEW

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.installer.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: BraveSoftware.BraveBrowser
-PackageVersion: 98.1.36.105
+PackageVersion: 98.1.36.106
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.installer.yaml:5]:
```suggestion
PackageVersion: 99.1.36.106
```

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.locale.en-US.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: BraveSoftware.BraveBrowser
-PackageVersion: 98.1.36.105
+PackageVersion: 98.1.36.106
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 99.1.36.106
```

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: BraveSoftware.BraveBrowser
-PackageVersion: 98.1.36.105
+PackageVersion: 98.1.36.106
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.yaml:5]:
```suggestion
PackageVersion: 99.1.36.106
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.yaml:5]: image

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.locale.en-US.yaml:5]: image

File: manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/BraveBrowser/98.1.36.106/BraveSoftware.BraveBrowser.installer.yaml:5]: image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**